invenio_rdm_records/services/review/service.py

```diff
@@ -193,7 +193,7 @@ def _submit(
         # Get record and check permission
         self.require_permission(identity, "update_draft", record=draft)
 
-        assert "restricted" in CommunityAccess.VISIBILITY_LEVELS
+        assert CommunityAccess.validate_visibility_level("restricted")
```

Comment:
Minor: Can we use the newly created enum? (same for line 198)
```suggestion
        assert CommunityAccess.validate_visibility_level(VisibilityEnum.RESTRICTED)
```

Reply:
This will always be true, what I wanted to check is if the string I will use in the query/business logic is part of the enum.

Reply:
Right! Thanks for explaining and sorry for the stupid comment :facepalm: